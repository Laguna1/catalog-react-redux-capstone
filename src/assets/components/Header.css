.header {
  min-height: 5%;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  font-size: calc(10px + 2vmin);
  color: orange;
  background-color: yellow;
  box-shadow: 1px 5px 9px 2px #d1d1d1;
  transition: box-shadow 0.5s, border 1s;
}

h2 {
  text-shadow: 2px 2px 5px red;
}

a {
  color: orange !important;
}

.btn-link {
  color: orange !important;
}

.btn:visited {
  text-decoration: none !important;
}

.nav-right {
  color: orange;
}
