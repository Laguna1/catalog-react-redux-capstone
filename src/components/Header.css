.header {
  background-color: greenyellow;
  min-height: 5%;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  font-size: calc(10px + 2vmin);
  color: white;
  border-bottom: 2px solid green;
}
