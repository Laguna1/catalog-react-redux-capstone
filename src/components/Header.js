import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <div className="header">
    <h1>Header logo here!</h1>
    <Link to="/" className="nav-link">All Meals</Link>
    <Link to="/" className="nav-link">Contacts</Link>
    <Link to="/" className="nav-link">About</Link>
  </div>
);

export default Header;
