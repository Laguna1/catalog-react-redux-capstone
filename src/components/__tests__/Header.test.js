import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Header from '../Header';

const renderer = new ShallowRenderer();

it('should render Header correctly', () => {
  renderer.render(<Header />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
