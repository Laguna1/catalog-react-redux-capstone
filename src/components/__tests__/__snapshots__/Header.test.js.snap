// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<Navbar
  className="header d-flex flex-row justify-content-between"
  collapseOnSelect={false}
  expand={true}
  variant="light"
>
  <div
    className="text-uppercase ml-5 font-weight-bolder mt-1"
  >
    <h2
      className="display-4"
    >
      Recipes-4-u
    </h2>
  </div>
  <div>
    <Link
      to="/"
    >
      <button
        className="px-5 btn btn-lg btn-link rounded-0 text-uppercase h5"
        onClick={[Function]}
        type="button"
      >
        Home
      </button>
    </Link>
    <span
      className="nav-right mx-3 text-uppercase h5"
    >
      About
    </span>
    <span
      className="nav-right mx-3 text-uppercase h5"
    >
      My recipes
    </span>
  </div>
</Navbar>
`;
